testbeam_analysis/tools/plot_utils.py

```diff
@@ -663,80 +695,78 @@ def plot_track_chi2(chi2s, fit_dut, output_pdf=None):
         x_limits = [1]
     x_limits.append(2500)  # plot fixed narrow range
     for x_limit in x_limits:
-        plt.clf()
-        plt.hist(chi2s, bins=100, range=(0, x_limit))
-        plt.xlim(0, x_limit)
-        plt.grid()
-        plt.xlabel('Track Chi2 [um*um]')
-        plt.ylabel('#')
-        plt.yscale('log')
-        plt.title('Track Chi2 for DUT %d tracks' % fit_dut)
-        if isinstance(output_pdf, PdfPages):
-            output_pdf.savefig()
-        elif output_pdf is True:
-            plt.show()
+        fig = Figure()
+        _ = FigureCanvas(fig)
+        ax = fig.add_subplot(111)
+        ax.hist(chi2s, bins=100, range=(0, x_limit))
+        ax.set_xlim(0, x_limit)
+        ax.grid()
```

Comment:
Was also wondering why, but it is indeed `Axes.grid()`: http://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.grid.html#matplotlib.axes.Axes.grid